src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -262,24 +263,14 @@ void DirectoryService::UpdateMyDSModeAndConsensusId()
         return;
     }
 
-    // If I was DS primary, now I will only be DS backup
-    if (m_mode == PRIMARY_DS)
+    // Check if I am the oldest backup DS (I will no longer be part of the DS committee)
```

Comment:
This comment applies to the code further down

Reply:
noted.